Source/Laser/LaserProfilesImpl/LaserProfileFieldFunction.cpp

```diff
@@ -37,14 +37,18 @@ WarpXLaserProfiles::FieldFunctionLaserProfile::init (
     for (auto const& s : symbols) { // make sure there no unknown symbols
         amrex::Abort("Laser Profile: Unknown symbol "+s);
     }
+
+    m_gpu_parser.reset(new ParserWrapper<3>(m_parser));
```

Comment:
Just a suggestion: we could consider this new function available since C++14 
```suggestion
    m_gpu_parser = std::make_unique< ParserWrapper<3> >(m_parser);
```

Reply:
Sure.

Reply:
That's no problem, as long as we are ok that the old and the new `ParserWrapper` can live for a short moment at the same time (the old one will be deleted after the new one is constructed).